config.mk

```diff
@@ -14,7 +14,7 @@ SBINDIR = $(PF_PREFIX)/sbin
 # Golang
 #
 GOVERSION = go1.13.1
-GOBINARIES = pfhttpd pfdhcp pfdns pfstats pfdetect galera-autofix pfacct pfcertmanager
+GOBINARIES = pfhttpd pfdhcp pfdns pfstats pfdetect galera-autofix pfacct pfcertmanager pfmaint
```

Comment:
I guess pfmaint is the new pfmon but that's clearly not a good name, it will be easily confused with pf-maint.pl

I think the name pfmon should still be kept

Reply:
I agree on this one, I was surprised to see it renamed to pfmaint - I strongly prefer pfmon

Reply:
pfmon name did not make sense to me since it is for maintenance tasks not monitoring.
However, I do agree that it should it could be confused for pf-maint.pl.
Maybe pfcron would be a better name since make more sense of it's purpose.

Reply:
pfcron sounds like a good name for me

Reply:
Me too